Source/WebCore/Modules/speech/SpeechRecognitionResultList.cpp

```diff
@@ -0,0 +1,33 @@
+
+#include "config.h"
+
+#if ENABLE(SPEECH_RECOGNITION)
+
+#include "SpeechRecognitionResultList.h"
+
+namespace WebCore {
+PassRefPtr<SpeechRecognitionResultList> SpeechRecognitionResultList::create(const Vector<RefPtr<SpeechRecognitionResult> >& results)
+{
+    LOG(SpeechRecognition, "%s:%s:%d \n\n", __FILE__, __func__, __LINE__);
+
+    return adoptRef(new SpeechRecognitionResultList(results));
```

Comment:
I believe PassRef is slowly not being used in WebKit upstream anymore. e.g. Remove PassRef.h after r177259
https://bugs.webkit.org/show_bug.cgi?id=160348

Maybe consider using a "&&"?